crypto/digestible/src/lib.rs

```diff
@@ -446,6 +446,25 @@ impl DigestibleAsBytes for [u8; 32] {}
 
 impl<Length: ArrayLength<u8>> DigestibleAsBytes for GenericArray<u8, Length> {}
 
+// Implementation for tuples of Digestible
+// This is treated as an Agg in the abstract structure hashing schema,
+// because that is how digestible-dreive handles tuple structs and enums in
+// tuples.
+
+impl<T: Digestible, U: Digestible> Digestible for (&T, &U) {
+    #[inline]
+    fn append_to_transcript<DT: DigestTranscript>(
+        &self,
+        context: &'static [u8],
+        transcript: &mut DT,
+    ) {
+        transcript.append_agg_header(context, b"Tuple");
+        self.0.append_to_transcript(b"0", transcript);
+        self.1.append_to_transcript(b"1", transcript);
+        transcript.append_agg_closer(context, b"Tuple");
+    }
+}
```

Comment:
this part is new in the latest commit, it is added to make it easy to digest `BTreeMap`

I should add some tests of digesting `BTreeMap`